Study_In_Germany.md

```diff
@@ -0,0 +1,28 @@
+# How the German higer education system works 😄
+
+### University
+
+### Technical University
+
+### University of applied science
+
+### Private University
+
+# What do you need to apply for Bachelor?
+
+## General
+
+1. German language competence: C1 or equivalent level in TestDaf
+2.
+
+## Applying after graduation of high school
+
+## Applying after graduation of University / college
+
+## Applying after 4th Semester of University
+
+# Please keep this in mind!
```

Comment:
```suggestion
# Being expelled after failing three mandatory exams
```
I think making the title of the section descriptive is better - you need to read through the contents